packages/driver/cypress/integration/commands/net_stubbing_spec.ts

```diff
@@ -1605,28 +1606,30 @@ describe('network stubbing', { retries: 2 }, function () {
         .wait('@err', { timeout: 50 })
       })
 
-      it('can timeout in req.reply handler', {
-        defaultCommandTimeout: 50,
-      }, function (done) {
+      it('can timeout in req.reply handler', function (done) {
         cy.on('fail', (err) => {
+          Cypress.config('defaultCommandTimeout', 5000)
           expect(err.message).to.match(/^A response callback passed to `req.reply\(\)` timed out after returning a Promise that took more than the `defaultCommandTimeout` of `50ms` to resolve\./)
 
           done()
         })
 
         cy.route2('/timeout', (req) => {
+          Cypress.config('defaultCommandTimeout', 50)
+
           req.reply(() => {
             return Promise.delay(200)
           })
-        }).visit('/timeout', { timeout: 500 })
+        })
+        .visit('/timeout')
       })
 
       it('can timeout when retrieving upstream response', {
         responseTimeout: 25,
       }, function (done) {
         cy.once('fail', (err) => {
           expect(err.message).to.match(/^`req\.reply\(\)` was provided a callback to intercept the upstream response, but the request timed out after the `responseTimeout` of `25ms`\./)
-          .and.contain('ESOCKETTIMEDOUT')
+          .and.match(/ESOCKETTIMEDOUT|ETIMEDOUT/)
```

Comment:
if CPU load is high, the socket can take more than 25ms to establish, which will result in an ETIMEDOUT instead of the ESOCKETTIMEDOUT